vhdbuilder/packer/pre-install-dependencies.sh

```diff
@@ -50,6 +50,9 @@ systemctlEnableAndStart sync-container-logs.service || exit 1
 
 # First handle Mariner + FIPS
 if [[ ${OS} == ${MARINER_OS_NAME} ]]; then
+  dnf_makecache || exit $ERR_APT_UPDATE_TIMEOUT
```

Comment:
I wonder if this should be after the fips install is done?

Reply:
+ @dmcilvaney @rmhsawyer for their insight.

For context:
We are placing a call to dnf_makecache and dnf_update in pre-install-dependencies to trigger updating to the latest kernel before the reboot.

Question:
Should we wait until FIPS is enabled and the FIPS related grubby and dracut-fips packages are installed before updating?

Reply:
@dmcilvaney or @rmhsawyer to comment

Is there a risk to installing an updated kernel before installingFIPS _and_ before rebooting into said newly updated kernel?

Reply:
Generally all of Mariner's kernels and openssl are fips 'compliant' (certified means a specific version of the .rpm). So long as the grub cmdline is set correctly, and the dracut-fips package remains installed I don't see any issues with how we handle the kernel etc. packages.